stan/math/prim/scal/prob/neg_binomial_lcdf.hpp

```diff
@@ -121,7 +121,7 @@ namespace stan {
           ops_partials.edge1_.partials_[i] += g1 / Pi;
         }
         if (!is_constant_struct<T_inv_scale>::value)
-          ops_partials.edge2_.partials_[i]  += d_dbl * pow(1-p_dbl, n_dbl)
+          ops_partials.edge2_.partials_[i]  += d_dbl * pow(1 - p_dbl, n_dbl)
             * pow(p_dbl, alpha_dbl-1) / beta_func / Pi;
```

Comment:
You missed the space around the operator in line 125.  Also, efficiency of multiple division here.